packages/main/src/RangeSlider.js

```diff
@@ -194,10 +214,15 @@ class RangeSlider extends SliderBase {
 	_onfocusout(event) {
 		if (this._isFocusing()) {
 			this._preventFocusOut();
-		} else {
-			this._setAffectedValue(null);
-			this._setInitialValue("startValue", null);
-			this._setInitialValue("endValue", null);
+			return;
+		}
+
+		this._setAffectedValue(null);
+		this._setInitialValue("startValue", null);
+		this._setInitialValue("endValue", null);
+
+		if (this.showTooltip) {
+			this._tooltipVisibility = this._constants().TOOLTIP_HIDDEN;
```

Comment:
Maybe it will be better if you use just a static method in the SliderBase class for storing the possible tooltips visibility state constants instead of adding an instance method, like: 

SliderBase:
```
static get TOOLTIP_STATES() {
		return {
	        TOOLTIP_VISIBLE: "visible",
	        TOOLTIP_HIDDEN: "hidden",
		};
	}
```
then here `SliderBase.TOOLTIP_STATES.TOOLTIP_HIDDEN`

Reply:
We have already discussed this with Martin and this was suggested initially, however it was decided to leave it as non static variable. We can briefly discuss it again and based on the outcome I will revise the code.

Reply:
Done